### PR TITLE
Update Modrinth Minotaur

### PR DESCRIPTION
**Pull request by wafflecoffee**
I know @PotatoPresident said he wanted to switch to mc-publish at one point or another, but the switch away from 1.x is a rather urgent one considering dependencies in 1.x are not and have never been supported. As it stands right now, your Minotaur configuration is resulting in dependencies being listed incorrectly. I've gone through and fixed the dependencies on the versions already on the site, but please do not make a new release without first merging this or switching to mc-publish.

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -218,7 +220,7 @@ curseforge {
             mainArtifact(tasks["remapJar"])
 
             relations(closureOf<CurseRelation> {
-                props["cfReqDeps"].toString().split(",").forEach {
+                props["dependencies"].toString().split(",").forEach {
```
Comment (PotatoPresident) [inline build.gradle.kts:223]: Why did you change the cf one? It seems to work.

Comment (wafflecoffee) [reply build.gradle.kts:223]: It's the same thing for both CF and MR, so there's no point in having the property named `cfReqDeps` when it's being used for MR too.

Comment (PotatoPresident) [reply build.gradle.kts:223]: Yeah I just realized that.
Semi related, why doesn't that work on the front end??

**Review by PotatoPresident — APPROVED** [review]

File: build.gradle.kts (hunk 1 above)
Comment (wafflecoffee) [reply build.gradle.kts:223]: Does it not?
https://streamable.com/ao31d3

**Comment by PotatoPresident** [comment]
Build failed. I thought my dev env was just broken. I'm not crazy! No idea what's wrong though.

File: build.gradle.kts (hunk 1 above)
Comment (PotatoPresident) [reply build.gradle.kts:223]: Well now I feel crazy again. I swear there was a message that said only project ID

**Comment by wafflecoffee** [comment]
No clue either - I don't know jack-anything about Kotlin, Netty, or Ledger. :smile:
